backend/src/packages/auth/auth.service.ts

```diff
@@ -31,6 +34,29 @@ class AuthService {
     return await this.userService.create(userRequestDto);
   }
 
+  public async verifyLoginCredentials({
+    email,
+    password,
+  }: UserSignInRequestDto): Promise<UserSignInResponseDto | null> {
+    const user = await this.userService.findByEmail(email);
+    const userWithPassword = await this.userService.getUserInfoWithPassword(
+      email,
+    );
```

Comment:
we need only one query, and then call UserEntity.initialize with it
```suggestion
    const user = await this.userService.getUserInfoWithPassword(
      email,
    );
```

Reply:
Yes, changed it!